app/src/main/java/com/ataulm/artcollector/HarvardArtMuseumApi.kt

```diff
@@ -76,6 +76,8 @@ data class ApiObjectRecord(
         @Json(name = "id") val id: Int,
         @Json(name = "title") val title: String,
         @Json(name = "description") val description: String?,
+        @Json(name = "url") val url: String,
```

Comment:
url ought to always be there since it seems to be a template with the objectid.

the creditline is sometimes missing I've noticed. in this case, we'll fallback to "Harvard Art Museums" though the place this fallback is done is incorrect.